app/controllers/dashboards_controller.rb

```diff
@@ -15,4 +15,12 @@ def show
     @feed = Dashboard.display_feed
   end
 
+  def send_flow
+    flow = Flowdock::Flow.new(:api_token => ENV['LP_FLOW'],
+        :source => "LaunchPairs", :from => {:name => current_user.first_name, :address => current_user.email})
```

Comment:
Line is too long. [110/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.